public/elements/zero-featured-streamer.html

```diff
@@ -101,8 +101,14 @@
       },
 
       ready: function() {
-        $.getJSON('https://api.twitch.tv/kraken/users/' + this.streamer,
-            this._handleStreamerInfoReceived.bind(this));
+        $.ajax({
+         type: 'GET',
+         url: 'https://api.twitch.tv/kraken/users/' + this.streamer,
+         headers: {
+           'Client-ID': '9ob60m56gzoz9ydsq6d6aeqp7w1ckqp'
```

Comment:
You should probably store this value in some global variable instead of replicating it.